pkgs/games/re3/default.nix

```diff
@@ -0,0 +1,42 @@
+{ branch ? "re3"
+, callPackage
+, fetchFromGitHub
+}:
+
+{
+  re3 = callPackage ./base.nix rec {
+    pname = "re3";
+    version = "unstable-2021-07-12";
+
+    src = fetchFromGitHub {
+      repo = "re3";
+      owner = "GTAmodding";
+      rev = "a4b92fe9bec17ad00dc436574f57844c4b8d49f5";
+      sha256 = "5CPgDhUgkB0B5EApwkdhh7qqEbtCE3Fee1CKu3BTGz4=";
+    };
+  };
+
+  miami = callPackage ./base.nix rec {
+    pname = "reVC";
+    version = "unstable-2021-07-12";
+
+    src = fetchFromGitHub {
+      repo = "re3";
+      owner = "GTAmodding";
+      rev = "523b23339cd761d934501dcbcc059d131b5a28c4";
+      sha256 = "Cb9C9Gh1VGPPwF2uO82wOdxD2rfaixdkBsClx3OFNvU=";
+    };
+  };
+
+  lcs = callPackage ./base.nix rec {
+    pname = "reLCS";
+    version = "unstable-2021-07-12";
+
+    src = fetchFromGitHub {
+      repo = "re3";
+      owner = "GTAmodding";
+      rev = "6f0f9d7d695056575120ff49cc01297419c0d0e4";
```

Comment:
Can we use the same rev for all packages?

Reply:
No, different branches mean different commits

Reply:
Can you add the branch information in a comment before the rev?

Reply:
Now I think this is clearer with the real branch names